controllers/machinehealthcheck_targets_test.go

```diff
@@ -337,68 +331,23 @@ func TestHealthCheckTargets(t *testing.T) {
 func TestRemediate(t *testing.T) {
 	namespace := "test-mhc"
 	clusterName := "test-cluster"
-	labels := map[string]string{"cluster": clusterName, "machine-group": "foo"}
-
-	machineSetORs := []metav1.OwnerReference{{APIVersion: clusterv1.GroupVersion.String(), Kind: "MachineSet"}}
-	kubeadmControlPlaneORs := []metav1.OwnerReference{{APIVersion: controlplanev1.GroupVersion.String(), Kind: "KubeadmControlPlane"}}
 
 	workerNode := newTestNode("worker-node")
-	workerMachine := newTestMachine("worker-machine", namespace, clusterName, workerNode.Name, labels)
-	workerMachine.SetOwnerReferences(machineSetORs)
-	workerMachineUnowned := newTestMachine("worker-machine", namespace, clusterName, workerNode.Name, labels)
-
-	controlPlaneNode := newTestNode("control-plane-node")
-	if controlPlaneNode.Labels == nil {
-		controlPlaneNode.Labels = make(map[string]string)
-	}
-	controlPlaneNode.Labels[nodeControlPlaneLabel] = ""
-
-	controlPlaneMachine := newTestMachine("control-plane-machine", namespace, clusterName, controlPlaneNode.Name, labels)
-	controlPlaneMachine.SetOwnerReferences(kubeadmControlPlaneORs)
-	if controlPlaneMachine.Labels == nil {
-		controlPlaneMachine.Labels = make(map[string]string)
-	}
-	controlPlaneMachine.Labels[clusterv1.MachineControlPlaneLabelName] = ""
+	workerMachine := newTestMachine("worker-machine", namespace, clusterName, workerNode.Name, nil)
 
 	testCases := []struct {
-		name          string
-		node          *corev1.Node
-		machine       *clusterv1.Machine
-		expectErr     bool
-		expectDeleted bool
-		expectEvents  []string
+		name            string
+		node            *corev1.Node
+		machine         *clusterv1.Machine
+		expectErr       bool
+		expectAnnotated bool
 	}{
 		{
-			name:          "when the machine is not owned by a machineset",
-			node:          workerNode,
-			machine:       workerMachineUnowned,
-			expectErr:     false,
-			expectDeleted: false,
-			expectEvents:  []string{},
-		},
-		{
-			name:          "when the node is a worker with a machine owned by a machineset",
-			node:          workerNode,
-			machine:       workerMachine,
-			expectErr:     false,
-			expectDeleted: true,
-			expectEvents:  []string{EventMachineDeleted},
-		},
-		{
-			name:          "when the node is a control plane node",
-			node:          controlPlaneNode,
-			machine:       controlPlaneMachine,
-			expectErr:     false,
-			expectDeleted: false,
-			expectEvents:  []string{},
-		},
-		{
-			name:          "when the machine is a control plane machine",
-			node:          nil,
-			machine:       controlPlaneMachine,
-			expectErr:     false,
-			expectDeleted: false,
-			expectEvents:  []string{},
+			name:            "it annotates the machine with the unhealthy annotation",
```

Comment:
Would be good to add another test case here that has existing annotations on the machine and check that any existing annotations are not dropped as part of this annotation process